Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -185,7 +185,7 @@
             "label": "Virtual application",
             "defaultValue": "",
             "required": false,
-            "visibleRule": "WebAppKind != webAppLinux && WebAppKind != webAppContainer && WebAppKind != \"\"",
+            "visibleRule": "WebAppKind != webAppLinux && WebAppKind != webAppContainer && WebAppKind != functionApp && WebAppKind != \"\"",
```

Comment:
Can you please visible rule for Post deployment action too? (hiding it for funciton apps)